resources/main.js

```diff
@@ -16,6 +17,25 @@ $(function () {
             const value = event.target.value;
             const textInput = document.getElementById(el.id.replace(/-dropdown$/, ''));
             textInput.value = value;
-        }
+        };
     });
-});
\ No newline at end of file
+  });
+
+$(function () { document.getElementById("mw-scratch-confirmaccount-click-copy").onclick = function() {
+  copyToClipboard(document.getElementById("mw-scratch-confirmaccount-verifcode"));
+  }
+});
+
+function copyToClipboard(temptext) {
+        var tempItem = document.createElement('textarea');
+        tempItem.value = temptext.innerText;
+        tempItem.style.top = "0";
+        tempItem.style.left = "0";
+        tempItem.style.position = "fixed";
+        document.body.appendChild(tempItem);
+        tempItem.focus();
+        tempItem.select();
+        document.execCommand('copy');
+        tempItem.parentElement.removeChild(tempItem);
+        alert(mw.message( 'scratch-confirmaccount-click-copy-alert').plain());
```

Comment:
Please use a [bubble notification](https://www.mediawiki.org/wiki/Bubble_notifications) for this.